metadata-models-custom/README.md

```diff
@@ -323,18 +357,22 @@ The full example can be found in [`CustomDataQualityRulesMCPSideEffect.java`](sr
 ```java
 public class CustomDataQualityRulesMCPSideEffect extends MCPSideEffect {
```

Comment:
In general I think links to code should be preferred for things that actually exist in the codebase to reduce the need to keep them up to date.

Reply:
True, but it makes for a better documentation experience to see the code inline in an abbreviated way. I think that once the interface stabilizes this will be pretty static.